Examples/ReactiveExample/ReactiveExample/API.swift

```diff
@@ -21,22 +21,24 @@ public struct API {
                 method: .post,
                 baseURL: baseURL,
                 path: "/api/v1/auth/login",
-                body: [
+                body: .encodable([
                     "username": username,
                     "password": password
-                ], shouldStub: true,
-                stub: StubResponse(statusCode: 200, fileName: "authresponse.json", delay: 3))
+                ]))
+//            , shouldStub: true,
```

Comment:
is the "," in the next line (comment) intended here to hide it from swiftlint because it would be the last item?

Reply:
Good point, I just forgot to clean up the code